### PR TITLE
Add hedvigers block

### PR DESCRIPTION
**Pull request by Iamsamwhoami**
This one goes into a feature branch :)

### CODE REVIEW

**Comment by hedvigoscar** [comment]
Looks like netlify cant build it though, might want to check that out ;)

**Comment by Iamsamwhoami** [comment]
@hedvigoscar did a lot of changes and added some snapshot tests and whatnot, could you please have a look once again :)

File: src/sections/about-us/hedvigers/desktop/selected-image.tsx [hunk 1]
```diff
@@ -73,17 +76,17 @@ export const SelectedImage: React.SFC<SelectedImageProps> = ({
...
-            <ImageContainer status={status}>
-              <Image src={image} />
-            </ImageContainer>
-          )}
-        </Transition>
-      ))}
-    </TransitionGroup>
-  </Container>
-);
+    <Container>
```
[the remark this answers is not in the document]
Comment (Iamsamwhoami) [reply src/sections/about-us/hedvigers/desktop/selected-image.tsx]: @hedvigoscar 🙈idk, but I've fixed that now

File: src/sections/about-us/hedvigers/desktop/selected-image.tsx [hunk 2]
```diff
@@ -35,10 +35,13 @@ const exit = keyframes({
   },
 });
 
-const getTransform = (status) => {
+const getTransform = (status: string) => {
```
[the remark this answers is not in the document]
Comment (Iamsamwhoami) [reply src/sections/about-us/hedvigers/desktop/selected-image.tsx]: @hedvigoscar def, though `TransitionStatus` wasn't exported, so I had to duplicate it in the file, fix is here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28649.

File: src/sections/about-us/hedvigers/desktop/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
```
[the remark this answers is not in the document]
Comment (Iamsamwhoami) [reply src/sections/about-us/hedvigers/desktop/index.test.tsx]: @hedvigoscar yes! and that's sorted!

**Comment by Iamsamwhoami** [comment]
I am merging this in, let's discuss in the final PR if we need :)

**Pull request merged**